### ISSUE TITLE
I want to add a mailto-link as an item

### ISSUE THREAD

**Issue by create-martin** [moodle-an-hochschulen/moodle-theme_boost_union#702]
**Is your feature request related to a problem? Please describe.**

when adding an item as static link, I tried to enter a mailto:-Link. But upon saving, the input field becomes empty again and I cannot save.

**Describe the solution you'd like**
Add your user story - a clear and concise description of what you want to happen.
e.g.: 
  - As a teacher I would like ...
  - As a student I would like ...

**Describe alternatives you've considered**

**Additional context**

**Comment by wiebkemueller-hsh** [comment]
Hello @create-martin could you provide some more context as to where is this happening? Smart menu? Regards Wiebke

**Comment by create-martin** [comment]
yes:) sorry. Smart Menu. I create a new item and wanted to add a mailto-Link --> e.g. Contact us and it opens an pre-defined e-mail
